pages/docs/manual/latest/function.mdx

```diff
@@ -157,6 +157,27 @@ Because we _do_ supply the unit OCaml knows we deliberately omit the `radius` pa
 let circle = drawCircle(~color, ());
 ```
 
+### Signatures and Type Annotations
+
+Functions with optional labeled arguments can be confusing when it comes to signature and type annotations. Indeed, an optional labeled argument is seen as wrapped in an `option` inside the function body, but is called unwrapped but when you apply the function.
```

Comment:
Quick rewording suggestion: 
> ... annotations. Indeed, they type of an optional labeled argument looks different depending on whether you're calling the function, or working inside the function body. Outside the function, a raw value is either passed in (`int`, for example), or left off entirely. Inside the function, the parameter is always there, but its value is an option (`option(int)`). This means that the type _signature_ is different, depending on whether you're writing out the _function_ type, or the _parameter type annotation_. The first being a raw value, and the second being an option.

Reply:
thanks @mrmurphy, I reworded it accordingly